connection_pool/connection_pool.go

```diff
@@ -0,0 +1,732 @@
+// Package with methods support master discovery feature
+// to work with a Tarantool cluster.
+//
+// Main features:
+//
+// -Return available connection from pool according to round-robin strategy.
+//
+// -Automatic master discovery by `mode` parameter.
```

Comment:
We decided not to use apostrophes in documentation lines because it's not processed by godoc.

Reply:
Fixed.